modules/nodes-pods-configuring-replication.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-configuring.adoc
+
+[id='nodes-pods-configuring-replication_{context}']
+= Understanding how to control pod counts using replication controllers
```

Comment:
i wonder if we should even document this.  customer should never use replication controllers or replicasets directly.  they will use deploymentconfigs and deployments.